classes/ActionScheduler_Versions.php

```diff
@@ -115,6 +115,58 @@ public static function initialize_latest_version() {
 	 * @return string
 	 */
 	public function active_source() {
+		$file   = __FILE__;
+		$dir    = __DIR__;
```

Comment:
Instead of assigning the value of these constants to variables (especially since we don't subsequently mutate those variables), we could just use the constants directly later on.

Reply:
I have these because it makes it easier to test with. But you're right, they can easily be removed.